src/window/mod.rs

```diff
@@ -45,6 +45,12 @@ impl Window {
         }
     }
 
+    /// Get raw SDL context for external use (e.g. audio context).
+    #[cfg(feature = "with-sdl")]
+    pub fn context(&mut self) -> sdl2::Sdl {
```

Comment:
Let's add a `sdl_` prefix to make it clear that this is something SDL specific:
```suggestion
    pub fn sdl_context(&mut self) -> sdl2::Sdl {
```